packages/abi-gen/package.json

```diff
@@ -20,21 +20,21 @@
         "test:coverage": "nyc npm run test --all && yarn coverage:report:lcov",
         "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
         "test_cli": "yarn test_cli:run_mocha && yarn diff_contract_wrappers",
-        "test_cli:clean": "yarn clean:sol && rm -rf test-cli/**/output && rm -rf test-cli/typescript/lib",
+        "test_cli:clean": "rm -rf test-cli/output && rm -rf test-cli/test_typescript/lib",
         "test_cli:build": "tsc --project test-cli/tsconfig.json",
-        "test_cli:run_mocha": "mocha --require source-map-support/register --require make-promises-safe test-cli/typescript/lib/**/*_test.js --timeout 100000 --bail --exit",
+        "test_cli:run_mocha": "mocha --require source-map-support/register --require make-promises-safe test-cli/test_typescript/lib/**/*_test.js --timeout 100000 --bail --exit",
         "rebuild_and_test": "run-s build test",
         "test:profiler": "SOLIDITY_PROFILER=true run-s build run_mocha profiler:report:html",
         "test:trace": "SOLIDITY_REVERT_TRACE=true run-s build run_mocha",
         "clean:sol": "rm -rf test-cli/artifacts",
         "compile:sol": "sol-compiler",
         "watch:sol": "sol-compiler -w",
         "generate_contract_wrappers": "run-p gen_typescript gen_python",
-        "gen_typescript": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output ./test-cli/typescript/output --backend ethers",
-        "gen_python": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/Python/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/Python/partials/**/*.handlebars' --output ./test-cli/python/output --language Python",
+        "gen_typescript": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output ./test-cli/output/typescript --backend ethers",
+        "gen_python": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/Python/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/Python/partials/**/*.handlebars' --output ./test-cli/output/python --language Python",
         "diff_contract_wrappers": "run-p diff_typescript diff_python",
-        "diff_typescript": "./diff.sh ./test-cli/typescript/expected-output ./test-cli/typescript/output",
-        "diff_python": "./diff.sh ./test-cli/python/expected-output ./test-cli/python/output",
+        "diff_typescript": "./diff.sh ./test-cli/expected-output/typescript ./test-cli/output/typescript",
+        "diff_python": "./diff.sh ./test-cli/expected-output/python ./test-cli/output/python",
```

Comment:
Not essential, but we had talked about collapsing these 3 `diff` lines to just `"diff_contract_wrappers": "./diff.sh ./test-cli/expected-output ./test-cli/output",`.  Does that still make sense to you?  Or do you think there's a good reason to keep them separate?

Reply:
yes that makes sense! I just overlooked it, thanks for catching